xls/examples/cubic_bezier.x

```diff
@@ -0,0 +1,52 @@
+// Copyright 2021 Leonardo Romor <leonardo.romor@gmail.com>
```

Comment:
Can you make this copyright The XLS Authors? And add yourself to the bottom of the README.md file as a contributor?